parm/use_cases/met_tool_wrapper/EnsembleStat/EnsembleStat.conf

```diff
@@ -48,10 +48,11 @@ FCST_ENSEMBLE_STAT_INPUT_TEMPLATE = {init?fmt=%Y%m%d%H}/arw-???-gep?/d01_{init?f
 OBS_ENSEMBLE_STAT_POINT_INPUT_DIR = {INPUT_BASE}/met_test/out/ascii2nc
 OBS_ENSEMBLE_STAT_POINT_INPUT_TEMPLATE = precip24_{valid?fmt=%Y%m%d%H}.nc
 
-
 OBS_ENSEMBLE_STAT_GRID_INPUT_DIR = {INPUT_BASE}/met_test/data/sample_obs/ST4
 OBS_ENSEMBLE_STAT_GRID_INPUT_TEMPLATE = ST4.{valid?fmt=%Y%m%d%H}.24h
 
+#ENSEMBLE_STAT_ENS_MEAN_INPUT_DIR =
```

Comment:
Are these intended to be commented out? Or can they be set but just empty?

Reply:
It could just be left empty. I left it commented to make it more clear that it is optional and that users can remove it from their own use case config files.

Reply:
OK, just wanted to make sure there was no ill-effect to the use case CI or anything by having them commented out. Works for me.